commons/addons/cellar-vs-fsbuckets.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Choosing between File System Buckets and Cellar S3
+position: 3
+shortdesc: This will help you understand the differenc between Cellar and FS Buckets.
+tags:
+- addons
+keywords:
+- S3
+- amazon
+- Storage
+- file
+- files
+---
+
+You want to store files but you can't decide between FS Buckets and Cellar. This article
+will compare them: usage, pros/cons, pricing. Get ready for the ultimate comparison.
+
+Clever Cloud uses immutable throwable VMs.
```

Comment:
s/throwable/disposable/g

Reply:
Thanks, at the time I could not find the right word… disposable is exactly was I was looking for.